Wikiwho.py

```diff
@@ -540,7 +540,7 @@ def printRevision(revision):
                 #print word
                 #text = text + ' ' + unicode(word.value,'utf-8') + "@@" + str(word.revision) 
                 text.append(word.value)
-                authors.append(word.revision)
```

Comment:
# This line makes authors array use revision ids
# authors.append(word.revision)